apps/frontend/src/components/ui/card.tsx

```diff
@@ -36,44 +36,51 @@ const CardTitle = React.forwardRef<
   <h3
     ref={ref}
     className={cn(
-      "text-2xl font-semibold leading-none tracking-tight",
-      className
+      'text-2xl font-semibold leading-none tracking-tight',
+      className,
     )}
     {...props}
   />
-))
-CardTitle.displayName = "CardTitle"
+));
+CardTitle.displayName = 'CardTitle';
```

Comment:
this is a lot of noise?
Mind reverting these?

Reply:
It was not formatted before 
so formatted